CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # ZLUX Angular File Tree
 
 ## 0.11.0
-
+* Added the option to open a file in new browser tab
```

Comment:
I think this was last release' version of the FT. Can you please make it 0.12, update package.json, and tag a new release to be used in the Editor?

Reply:
Done